packages/xrpl/src/utils/stringConversion.ts

```diff
@@ -6,7 +13,7 @@
  * @category Utilities
  */
 function convertStringToHex(string: string): string {
```

Comment:
Should this be used in place of `bytesToHex(new TextEncoder().encode(string))` in more places in the code? (I think I saw instances of that in files I reviewed, but I am not sure if they were from within the xrpl package or other ones which wouldn't have access to this helper)

Reply:
Resolved in 11dd21c101084f6c00ee9d606bbdb0ad1dc048e7